code_examples/c++/helloWorld.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main(int argc, char** argv) {
```

Comment:
Should be `char* argv[]` according to http://en.cppreference.com/w/cpp/language/main_function